.github/workflows/cmake_ci.yml

```diff
@@ -55,3 +55,33 @@ jobs:
         working-directory: ./build
         run: |
           ctest -C ${{matrix.build_type}} --output-on-failure
+
+      - name: Set up Python
+        if: matrix.finufft_static_linking
```

Comment:
should this be if not static linking?

Reply:
> should this be if shared building?

It doesn't matter really. Only point is to not do two separate builds since this option has no bearing on the compilation (uses static linking by default, as far as I remember).

> As for python we can make the wheels when we make an .so?

The two build processes are separate (i.e., the one earlier when we call `cmake` and this one where we call `pip install`). I tried to make `pip install` use the existing build directory but ended up getting strange complaints about mismatching cmake caches, so I left it.